proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,133 @@
+
+We are confident that we have and continue to achieve increased ecosystem engagement with the Stargate testnet.
+
+### 2. Critical Partner Support
+
+We delivered critical partner support to leading ATOM exchanges. Out of the exchange support efforts, we documented the [IBC Readiness Matrix](https://github.com/cosmosdevs/stargate/blob/master/ibc_readiness_matrix.md) that outlines the levels of readiness that exchanges may select as they upgrade to Cosmos Stargate.
+
+We also completed and delivered a completed legacy Amino Audit, [AminoRest & You](https://github.com/cosmosdevs/stargate/blob/master/audit.md). The core finding of this audit was that changes to the underlying structs result in an interface that is close to the prior version allowing legacy queries to return valid data.
+
+### 3. Documentation and Self-Certification
+
+Documentation was a success for the Cosmos Stargate effort. The upgrades with the most breaking changes such as legacy Amino have a complete audit with documentation on exceptions for blockchain API interfaces.
+
+Most exchanges and wallets have taken self-certification on directly. Our team continues to provide real-time support on multiple partner slack channels and on the Cosmos [#stargate Discord](https://discord.gg/W8trcGV) channel.
+
+### Cosmos Stargate Integration Success
+
+Cosmos Stargate integration success with exchanges and wallet providers reflects the quality of the code developed by the various Cosmos teams.
+
+## Prop 29
+
+This upgrade also implements the fund recovery procedure defined in proposal 29. The code is [here](https://github.com/cosmos/gaia/blob/main/app/prop29.go). As requested during the discussion of the governance proposal, the migration and fund recovery procedure verifies all signatures. This code was tested during the cosmoshub-3 simulated upgrade testnet.ƒ
+
+## Stargate Upgrade Steps
+
+------------
+The upgrade steps for the simulated upgrade of the current Cosmoshub mainnet to Cosmos Stargate includes the following steps:
+
+This section is with the current `gaia 2.0.*` implementation.
+
+  1. Validators should set their `gaia` with a halt time of `1611813600`ie. 06:00 UTC on Jan 28th in Unix time.
+
+  2. Validators should then export the current cosmos state with `gaiad export > cosmoshub-3-export.json`
+
+  3. Validators should determine the height of the last block.
+
+  4. Validators should back up their `.gaiad` directory.
+
+This section is with the upgrade `gaia 3.0.*` implemenataion.
+
+  1. Validators should then migrate the exported genesis state. `gaiad migrate cosmoshub-3-export.json --chain-id=cosmoshub-4 --initial-height [last_cosmoshub-3_block+1] > genesis.json`
```

Comment:
why is this not a zero height upgrade? the last one upgrades reset

Reply:
We have committed to a number of partners mostly exchanges that we would not reset height during this upgrade. 

Resetting height introduces substantial complexities in their internal systems and we have communicated that this would be unnecessary.

Reply:
I also think zero height upgrades are a much worse UX for users, for both a conceptual front of how they think about the chain, and also for querying historical data. (They just want to query what happened at block x, they shouldn't have to know which hard fork it was in. Noone keeps track of which btc hard fork number we're on rn.)

Reply:
yeah I agree it makes the most sense. I'd just point that out on the gov proposal on a small note at the bottom as the previous upgrades started from 0.